examples/sample-article/sample-article.xml

```diff
@@ -980,19 +980,36 @@ the xsltproc executable.
                 <reading-questions>
                     <introduction>
                         <p>A set of reading questions may have an <tag>introduction</tag>, perhaps for preparatory explanation.</p>
+                        <p>If a student has logged in to the HTML version,
+                           then they can answer the reading questions directly in the book.
+                           Inline math LaTeX can be entered using
+                           <c>$</c>...<c>$</c> or <c>\(</c>...<c>\)</c> delimiters,
+                           and inline AsciiMath using backticks <c>`</c>...<c>`</c> as delimiters.
+                        </p>
                     </introduction>
 
                     <exercise xml:id="first-reading">
-                        <statement>
-                            <p>This is a <term>reading question</term> that you might have a student read prior to a class session, based on reading part of the book.</p>
-                        </statement>
-                        <answer>
-                            <p>Which might help make the classroom session improve.</p>
-                        </answer>
+                      <p>
+                        This is a <term>reading question</term> that you might have a student answer prior to a class session,
+                        based on reading part of the book.
+                        A quick glance before class can help you tailor class time to the specific needs of your students.
+                        The perfect reading question will reveal whether the student has read and understood the material,
+                        and will be difficult to answer if they have not.
+                        What do you think of that?
+                      </p>
                     </exercise>
 
                     <exercise>
-                        <p>And a second one, with a cross-reference to the first, as a check on numbering: <xref ref="first-reading" text="type-global"/>.</p>
+                      <p>
+                        And a second one, with a cross-reference to the first,
+                        as a check on numbering:
+                        <xref ref="first-reading" text="type-global"/>.
+                        Reading questions are allowed to have answers,
+                        but providing answers misses the point of a reading question,
+                        and the answer knowl interacts poorly with the mechanism used to
+                        allow students to answer directly in the book.
+                        Do you think the schema should ban answers to reading questions?
```

Comment:
No, I do not think it should be banned; some reading questions I assign often need a sample answer.  Example which folks may use:

> Give an example of a real-life situation where you experienced a nonzero second derivative, and explain.

Actual type of answer students give if I provide a sample:
> I noticed that as I was doing my weight training, the increase in the number of pounds I could lift got smaller each week, though I kept on improving.

Potential answer without a sample:
> I noticed that something looked like a parabola, and I know that $x^2$ has a nonzero second derivative.

However, suggesting that most reading questions not have a provided answer seems quite sound to me.

Reply:
We'll want answers in the source, so they can migrate to an Instructor's Manual. So the schema shouldn't ban them.  

But it should be an easy matter to ban them from appearing in the HTML conversion (or any other conversion) at the place where the question is born (and any `solutions` division as well.)  Once I get switches in place we can discuss which options to always turn off.